### ISSUE TITLE
[Bug][a11y] The delete bookmark sign from the "Edit bookmark" sections is not labeled

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#4435]
## Prerequisites
You have Talkback option enabled from your Android settings.

## Steps to reproduce
1. Navigate to any web page
2. Bookmark it
3. Tap "edit" from the "Bookmark saved" dialogue
4. Tap on the "Delete bookmark"![image](https://user-images.githubusercontent.com/51314259/62283301-ee067380-b459-11e9-831b-5378b571a01a.png) sign
![rsz_screenshot_20190801-121116666](https://user-images.githubusercontent.com/51314259/62283233-c1eaf280-b459-11e9-9403-c4ea03adae88.png)

### Expected behavior
Has a label

### Actual behavior
Has no label 

### Device information

* Android device: Google Pixel 3XL (Android 9)
* Fenix version: RC 1.2.0-rc.1 from 1/8

#### Notes
Simillar to #1392
